### ISSUE TITLE
Invalid JS can be generated for window.WPCOM_site_ID

### ISSUE THREAD

**Issue by paulschreiber** [Automattic/jetpack#2001]
If the site ID isn't defined, you can end up with invalid JS:

    window.WPCOM_site_ID = ;

This is a result of the work done in 55af9979. The PHP code in question is:
```php
 window.WPCOM_site_ID = <?php echo defined( 'IS_WPCOM' ) && IS_WPCOM ? get_current_blog_id() : Jetpack_Options::get_option( 'id' ); ?>;
```

**Comment by kraftbj** [comment]
The site ID should always be defined for a site connected to WP.com. I'm assuming you're using this in development mode, though?

**Comment by paulschreiber** [comment]
Yes, I'm using VIP Quickstart. Perhaps you could wrap the output in `wp_json_encode()`.

**Comment by jeherve** [comment]
This was fixed in 4178c9e92202401b83623ee1b5ab0fe3c3192ff9
